frontend/www/js/omegaup/arena/qualitynomination_popup.js

```diff
@@ -15,7 +15,8 @@ OmegaUp.on('ready', function() {
         props: {
           nominated: this.nominated,
           solved: this.solved,
-          dismissed: this.dismissed
+          dismissed: this.dismissed,
+          canNominateProblem: this.canNominateProblem
```

Comment:
```suggestion
          canNominateProblem: this.canNominateProblem,
```

igual en L60, así la siguiente vez que haya que agregar una propiedad no tenemos que volver a cambiar una línea no-relacionada.

Reply:
Listo